src/openapi-generator.ts

```diff
@@ -85,15 +88,22 @@ export class OpenAPIGenerator {
     component: OpenAPIComponentObject;
   }[] = [];
 
-  constructor(private definitions: OpenAPIDefinitions[]) {
+  constructor(
+    private definitions: OpenAPIDefinitions[],
+    private openAPIVersion: OpenApiVersion = '3.0.0'
```

Comment:
Since we are to make a major version with breaking changes. I feel like we can make this a mandatory field. How do you feel about that :thinking: